api/types/events/events.proto

```diff
@@ -1169,7 +1171,13 @@ message AppSessionStart {
         [ (gogoproto.nullable) = false, (gogoproto.embed) = true, (gogoproto.jsontag) = "" ];
 
     // PublicAddr is the public address of the application being requested.
+    // TODO: this information is also present on the AppMetadata, we can
+    // deprecate this field in the future.
     string PublicAddr = 7 [ (gogoproto.jsontag) = "public_addr" ];
```

Comment:
You can mark this field as deprecated using GRPC tag:

```suggestion
    string PublicAddr = 7 [ (gogoproto.jsontag) = "public_addr", deprecated = true ];
```

It won't do anything meaningful, just update the generated godoc to say that it's deprecated, but still useful sometimes to convey intent.